pom.xml

```diff
@@ -239,6 +223,11 @@ THE SOFTWARE.
         <artifactId>jcifs</artifactId>
         <version>1.3.17-kohsuke-1</version>
       </dependency>
+      <dependency>
+        <groupId>org.apache.sshd</groupId>
+        <artifactId>sshd-core</artifactId>
+        <version>1.2.0</version> <!-- TODO 1.3.0 requires Java 8 -->
```

Comment:
The client uses 1.2.0. The server could continue to run 0.14.0. I had problems making the `test` module handle two different versions of `sshd-core` in use. I will see if it is possible to resolve that at least temporarily.

Anyway all discussion of the pros or cons of upgrading should be kept in https://github.com/jenkinsci/sshd-module/pull/10.